_includes/footer.html

```diff
@@ -0,0 +1,56 @@
+<footer class="c-footer">
+  <div class="c-footer__container">
+    <section class="c-footer__award-container">
+      <img src="../assets/Group 2.svg" class="c-footer__award-image" alt="">
+      <p class="c-footer__award-text">named one of the best workplaces to work in ohio for 2016, 2017 and 2018</p>
+    </section>
+    <div class="c-footer__link-lists">
+      <ul class="c-footer__link-list">
+        <h1 class="c-footer__link-title">pilots</h4>
```

Comment:
This heading can be moved outside of the unordered list so the structure would look like this:
```
<h_>...</h_>
<ul>
   <li>....</li>
</ul>
```

Reply:
that semantically makes more sense 😄

Reply:
but it messes up my flexbox layout badly. I don't know if I have enough time to change this. I gotta keep moving because this is due tomorrow 😅

Reply:
I fixed it 😎